project/src/components/main-screen/main-screen.tsx

```diff
@@ -1,14 +1,21 @@
+import {useState} from 'react';
 import Header from '../header/header';
 import Locations from '../locations/locations';
 import OfferList from '../offer-list/offer-list';
-import {Offers} from '../../types/offers';
+import Map from '../map/map';
+import {Offer} from '../../types/offers';
 
 type MainScreenProps = {
-  cards: Offers;
+  cards: Offer[];
   offersCount: number;
 }
 
 function MainScreen({cards, offersCount}: MainScreenProps): JSX.Element  {
+  const [activeCard, setActiveCard] = useState<Offer | undefined>(undefined);
+  const handleCardHover = (offerId: number | undefined) => {
+    const currentOffer = cards.find((offer) => offer.id === offerId);
+    setActiveCard(currentOffer);
+  };
   return (
```

Comment:
```suggestion
  };
  
  return (
```

перед ретерн, если что-то есть, по хорошему бы пустую строку ставить